### DISCUSSION TITLE
Construction worker, recons and drill instructors?

### DISCUSSION THREAD

**Discussion by javierlovessylvie** [VJ-HLR-Developers/Half-Life-Resurgence#64] [Ideas]
I think these npcs should be added to the addon like the drill and recon beign neutral at player but friendly at hecu and construction workers be friendly to player and all black mesa npcs i would also say that forklift driver gus but he is just an actor so what do you think? :)

**Comment by justwatchinghere** [comment]
i feel like this cool especially sence im having hard time finding good models for them

**Comment by SerifLol** [comment]
Boost
